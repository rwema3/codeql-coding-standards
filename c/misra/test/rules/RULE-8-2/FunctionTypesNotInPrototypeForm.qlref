rules/RULE-8-2/FunctionTypesNotInPrototypeForm.ql