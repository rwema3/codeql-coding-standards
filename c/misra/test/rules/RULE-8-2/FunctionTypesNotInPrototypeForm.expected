| test.c:3:6:3:7 | f1 | Function f1 declares parameter that is unnamed. |
| test.c:4:6:4:7 | f2 | Function f2 does not specifiy void for no parameters present. |
| test.c:5:6:5:7 | f3 | Function f3 does not specifiy void for no parameters present. |
| test.c:7:5:7:6 | f5 | Function f5 declares parameter in unsupported declaration list. |
